ts/features/bonus/cgn/saga/index.ts

```diff
@@ -78,4 +90,25 @@ export function* watchBonusCgnSaga(bearerToken: string): SagaIterator {
     cgnGenerateOtp,
     backendCGN.generateOtp
   );
+
+  // CGN Offline Merchants
+  yield takeLatest(
```

Comment:
This effects are related to the `cgn_merchants_v2` flag, as mentioned in the PR description?
If yes could they be included in a FF check block?

Reply:
`cgn_merchants_v2` just switches between double tab view (for local and online merchants) and single list with both results merged in one single list of data, so answering the question no need to check the remote FF here